hudi-common/src/main/java/org/apache/hudi/common/util/collection/ExternalSpillableMap.java

```diff
@@ -272,7 +272,10 @@ public Collection<R> values() {
       return inMemoryMap.values();
     }
     List<R> result = new ArrayList<>(inMemoryMap.values());
```

Comment:
It would be great if u can also implement the `values()` function in `RocksDbDiskMap`

Reply:
currently  RocksDbDiskMap and BitCaskDiskMap both not support values function, so fix it here can be work

Reply:
Does different disk based map has especial implementations for the `values` method ? For example the RocksDB backed map can use a prefix search.

Reply:
seems good，but may support it first，otherwise would cause error in other moudule call it